src/shared/contexts/query/context.tsx

```diff
@@ -98,16 +103,49 @@ export const QueryContext =
   React.createContext<QueryContextType>(DEFAULT_CONTEXT)
 
 const buildQueryRequest = (
-  org,
+  org: Organization,
   text: string,
   override: QueryScope,
   options?: QueryOptions
-) => {
+): {
+  url: string
+  body: RequestBody | null
+  headers: {
+    'Content-Type': string
+    'Accept-Encoding': string
+    Accept?: string
+  }
+} => {
   const mechanism = options?.overrideMechanism ?? OverrideMechanism.Extern
```

Comment:
Will these changes (above) affect all queries, not just influxQL queries?

Reply:
I manually tested Flux and SQL for the above change and they won't affect other queries. And the CI tests all passed, so the changes are fine 👍 Thanks for asking!